src/engine/enginemaster.cpp

```diff
@@ -381,11 +382,25 @@ void EngineMaster::processChannels(int iBufferSize) {
     }
 }
 
+void EngineMaster::finishStartup() {
+    QThread::currentThread()->setObjectName("Engine");
+#ifdef __LINUX__
+    const auto cpuSet = CmdlineArgs::Instance().getEngineCpuSet();
```

Comment:
Even though this is only a one time issue, I would prefer not to use Instance() from the engine thread. 
Can we cache the result in a QAtomicInt and use it here?

Reply:
Wait, what is auto? Please don't use auto here. The type is out of sight.